backends/ebpf/CMakeLists.txt

```diff
@@ -78,9 +78,9 @@ add_dependencies(p4c_driver linkp4cebpf)
 
 # Tests
 
-set(EBPF_DRIVER_KERNEL "${CMAKE_CURRENT_SOURCE_DIR}/run-ebpf-test.py -t kernel")
-set(EBPF_DRIVER_BCC "${CMAKE_CURRENT_SOURCE_DIR}/run-ebpf-test.py -t bcc")
-set(EBPF_DRIVER_TEST "${CMAKE_CURRENT_SOURCE_DIR}/run-ebpf-test.py -t test")
+set(EBPF_DRIVER_KERNEL "${CMAKE_CURRENT_SOURCE_DIR}/run-ebpf-test.py -t kernel -u ${P4C_BINARY_DIR}")
+set(EBPF_DRIVER_BCC "${CMAKE_CURRENT_SOURCE_DIR}/run-ebpf-test.py -t bcc -u ${P4C_BINARY_DIR}")
+set(EBPF_DRIVER_TEST "${CMAKE_CURRENT_SOURCE_DIR}/run-ebpf-test.py -t test -u ${P4C_BINARY_DIR}")
```

Comment:
Why not pass the path to the compiler itself?

Reply:
I guess it can be the path to the compiler. Will change.